src/main/java/net/sacredlabyrinth/phaed/simpleclans/ClanPlayer.java

```diff
@@ -623,9 +623,10 @@ public String getPastClansString(String sep)
      */
     public Set<String> getPastClans()
     {
-        HashSet<String> pc = new HashSet<>();
-        pc.addAll(pastClans);
-        return pc;
+//        HashSet<String> pc = new HashSet<>();
+//        pc.addAll(pastClans);
+//        return pc;
+        return pastClans;
```

Comment:
How does this resolve anything? Is the caller of `getPastClans` mutating the returned Set?

Reply:
Exactly

Reply:
Hmm, and there's no proper way to mutate the set otherwise? Though I guess this is okay (and more memory-friendly, marginally) if there's no need for this to be immutable.

Reply:
I don't think it needs to be immutable.